### ISSUE TITLE
Can you delete this issue please ?

### ISSUE THREAD

**Issue by rohanjulka19** [magdalea446/So-strange-ppl-nowdays-#1]

**Comment by magdalea446** [comment]
Im not God😊... If someone choose to leave suddenly, why should i stop this?!

**Comment by magdalea446** [comment]
> Step - 1 Forgive the them
> Step - 2 Really Really forgive them
😟

**Comment by magdalea446** [comment]
😟

**Comment by rohanjulka19** [comment]
> Im not God... If someone choose to leave suddenly, why should i stop this?!

Well because you like the other person 😉

**Comment by magdalea446** [comment]
So funnny and so wrong ur thinking😂😂

**Comment by magdalea446** [comment]
> > Im not God... If someone choose to leave suddenly, why should i stop this?!
> 
> Well because you like the other person

So funnny and so wrong ur thinkingjoyjoy

**Comment by rohanjulka19** [comment]
> So funnny and so wrong ur thinking

Don't worry soon you mind will change. 😝

**Comment by magdalea446** [comment]
> > So funnny and so wrong ur thinking
> 
> Don't worry soon you mind will change.

It has never,,, 😟

**Comment by magdalea446** [comment]
> > > So funnny and so wrong ur thinking
> > 
> > 
> > Don't worry soon you mind will change.
> 
> It has never,,,

And never will... 😊

**Comment by magdalea446** [comment]
> > > So funnny and so wrong ur thinking
> > 
> > 
> > Don't worry soon you mind will change.
> 
> It has never,,,

And never will... 😊

**Comment by rohanjulka19** [comment]
Well 

> > > > So funnny and so wrong ur thinking
> > > 
> > > 
> > > Don't worry soon you mind will change.
> > 
> > 
> > It has never,,,
> 
> And never will...


> > > So funnny and so wrong ur thinking
> > 
> > 
> > Don't worry soon you mind will change.
> 
> It has never,,,


> > > > So funnny and so wrong ur thinking
> > > 
> > > 
> > > Don't worry soon you mind will change.
> > 
> > 
> > It has never,,,
> 
> And never will...

That's a lot of confidence in yourself.

**Comment by rohanjulka19** [comment]
Anyways would you like to move this conversation on a more convenient platform such as hike.

**Comment by magdalea446** [comment]
Yea bcs i know well about myself and i believe a lot on her😅

**Comment by magdalea446** [comment]
NAh, only using stardoll atm😅

**Comment by rohanjulka19** [comment]
> NAh, only using stardoll atm

You won't have have to share any personal information.

**Comment by rohanjulka19** [comment]
You want to talk to me right ?

**Comment by magdalea446** [comment]
I said what i had to say, now good night 😊

**Comment by rohanjulka19** [comment]
Just so you know I am very bad at understanding people.

**Comment by rohanjulka19** [comment]
And hints

**Comment by magdalea446** [comment]
Thats ur problem, i wish u will solve it😅

**Comment by rohanjulka19** [comment]
Huh ? How by overthinking all night ? 😂

**Comment by magdalea446** [comment]
If u try to open ur mind a lil bit there is no need to overthink all night(on my opinion😅)... Anyway, u know it...

**Comment by rohanjulka19** [comment]
Ok but please note that we can not communicate on GitHub or stardoll for that matter another platform which has ease of communication will be fine.

**Comment by magdalea446** [comment]
If u think like that, then dont comunicate there😊.. Go and comunicate on ur other platforms. Bye

**Comment by rohanjulka19** [comment]
Well thank you for being straight forward sorry it didn't work. Bye

**Comment by magdalea446** [comment]
Best wishes, now good bye for real.

**Comment by magdalea446** [comment]
Why?

**Comment by magdalea446** [comment]
And how😐

**Comment by magdalea446** [comment]
> _No description provided._

Have you blocked me?? 😐

**Comment by rohanjulka19** [comment]
Yes I did block you but now I have unblocked you . I want to forget this ever happened please refer to this link for deleting an issue.
https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/deleting-an-issue

**Comment by magdalea446** [comment]
To forget what?? 😐😂

**Comment by magdalea446** [comment]
Whatever, i see u become repeatedly angry with me without even knowing me, without any reason, if this makes u happy..then go on. But im not going to delete this issue bcs i didnt opened it. You opened it, u can delete it if u want.

**Comment by rohanjulka19** [comment]
I cannot delete this as you own this repo.

**Comment by rohanjulka19** [comment]
So please delete this issue

**Comment by magdalea446** [comment]
Anyway no need, bcs im going to delete my acc from here

**Comment by rohanjulka19** [comment]
Ok, sorry for acting rude or strange ..... Goodbye :)

**Comment by magdalea446** [comment]
Ok but how to delete it😐

**Comment by magdalea446** [comment]
I have 30 mins trying to but i cant find the way

**Comment by rohanjulka19** [comment]
click the icon on the top right corner 
click settings
go to accounts tab
scroll down 
click on delete your account

**Comment by magdalea446** [comment]
Ok found it... Thanks and sorry for disturbing you 😊

**Comment by rohanjulka19** [comment]
No problem 👍
